docs/source/example_input/boosted_frame_script.py

```diff
@@ -105,19 +109,17 @@ def dens_func( z, r ):
     n : 1d array of floats
         Array of relative density, with one element per macroparticles
     """
-    # Convert longitudinal coordinates from boosted frame to lab frame
-    # (NB: the density is Lorentz-transformed inside the Simulation object)
-    z = z*boost.gamma0
```

Comment:
I am wondering if this is not a bit dangerous, if a user wants to use a `dens_func` for a relativistic beam.

In that case, the user will probably copy the above function and forget to change 
```
z = z*boost.gamma0
```
to 
```
z = z/(boost.gamma0*(1+boost.beta0))
```

Therefore, I think I prefer `boost.static_length` and `boost.copropag_length`, where the assumptions are a bit more explicit. (Although the user might still use `boost.static_length` for a propagating bunch...)

Reply:
mh, I agree with your point but at the same time i think for a lot of cases it is more difficult for the user to adapt the density profile using only relative length scales. Imagine e.g. a case where a file-based density profile is loaded.

In general, I think it is very important to understand the transformations of the initial scales from the lab frame to the boosted frame. Once this is clear to the user, having z*boost.gamma0 as the transformation inside the density function can be more useful than just boosting e.g. the length of the up_ramp.

But I am okay with keeping the old definition.

Reply:
offline discussion

* add a comment that this is only true for a static plasma
* adapt documentation for input / boost converter